packages/plugin-list/src/ListHtmlDomRenderer.ts

```diff
@@ -11,10 +11,7 @@ export class ListHtmlDomRenderer extends AbstractRenderer<Node[]> {
     async render(node: ListNode): Promise<Node[]> {
         const tag = node.listType === ListType.ORDERED ? 'OL' : 'UL';
         const domListNode = document.createElement(tag);
-        const attributes = node.modifiers.find(Attributes);
-        if (attributes && attributes.constructor.name === 'Attributes') {
```

Comment:
Why the constructor name check ? Didn't you have your "exactMatch" code for this ?